operator_ui/src/pages/JobsIndex/JobsIndex.tsx

```diff
@@ -42,22 +42,38 @@ interface Job<T> {
 }
 
 export type DirectRequest = Job<models.JobSpec>
-export type OffChainReporting = Job<models.OcrJobSpec>
-export type CombinedJobs = DirectRequest | OffChainReporting
+export type JobSpecV2 = Job<models.JobSpecV2>
+export type CombinedJobs = DirectRequest | JobSpecV2
 
-function isDirectRequest(job: CombinedJobs): job is DirectRequest {
-  return job.type === JobSpecTypes.jobSpec
+function isJobSpecV1(job: CombinedJobs): job is DirectRequest {
+  return job.type === JobSpecTypes.v1
 }
 
-function isOffChainReporting(job: CombinedJobs): job is OffChainReporting {
-  return job.type === JobSpecTypes.ocrJobSpec
+function isJobSpecV2(job: CombinedJobs): job is JobSpecV2 {
+  return job.type === JobSpecTypes.v2
+}
+
+function isFluxMonitorJobSpecV2(job: JobSpecV2) {
+  return job.attributes.type === 'fluxmonitor'
+}
+
+function isOCRJobSpecV2(job: JobSpecV2) {
+  return job.attributes.type === 'offchainreporting'
 }
 
 function getCreatedAt(job: CombinedJobs) {
-  if (isDirectRequest(job)) {
+  if (isJobSpecV1(job)) {
     return job.attributes.createdAt
-  } else if (isOffChainReporting(job)) {
-    return job.attributes.offChainReportingOracleSpec.createdAt
+  } else if (isJobSpecV2(job)) {
+    const { fluxMonitorSpec, offChainReportingOracleSpec } = job.attributes
+
+    if (isFluxMonitorJobSpecV2(job)) {
+      return fluxMonitorSpec ? fluxMonitorSpec.createdAt : new Date().toString()
```

Comment:
Same Q about `createdAt` being undefined - is it possible in practice or just TS?